Build failure related to option.h

Hello,

We sporadically see build failures that terminate the build with this message:
```
.../cvc5/src/src/parser/tptp/tptp.cpp:24:10: fatal error: options/options.h: No such file or directory
   24 | #include "options/options.h"
      |          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
```

This happens in maybe 1 out of 10 builds, and only on a system with a large number of cores (256 cores). We use `make -j 256` on that machine.

The configuration is as described in #8631.

This has low priority for us as we simply avoid building cvc5 on that machine.

--- a/src/parser/tptp/tptp.cpp
+++ b/src/parser/tptp/tptp.cpp
@@ -21,8 +21,6 @@
 
 #include "api/cpp/cvc5.h"
 #include "base/check.h"
-#include "options/options.h"
-#include "options/options_public.h"
 #include "parser/parser.h"
 #include "smt/command.h"
 #include "theory/logic_info.h"

